plugins/modules/packaging/os/openbsd_pkg.py

```diff
@@ -42,6 +42,13 @@
             not already installed.
         type: bool
         default: no
+    snapshot:
+        description:
+          - Force C(%c) and C(%m) to expand to C(snapshots), even on a release
+            kernel.
+        type: bool
+        default: no
+        version_added: 1.2.0
```

Comment:
Since 1.2.0 has been released already:
```suggestion
        version_added: 1.3.0
```